generator.yml

```diff
@@ -962,7 +956,7 @@ builds:
   - {<< : *ppc64le_suse,      << : *next,             << : *clang,           boot: true,  llvm_version: *llvm_11}
   # https://github.com/ClangBuiltLinux/linux/issues/1023
   # https://github.com/ClangBuiltLinux/linux/issues/1143
```

Comment:
do these comments need to be deleted as well?

Reply:
Yes, good call!